<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cam_rh2v2_cfgdlg</class>
 <widget class="QDialog" name="cam_rh2v2_cfgdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Frame accumulation count (for preview only)</string>
         </property>
         <property name="buddy">
          <cstring>sbAccCnt</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbAccCnt">
         <property name="maximum">
          <number>4096</number>
         </property>
         <property name="value">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Number of frames to be read-out (acquisition only)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFDoubleEdit" name="sbFrameCnt"/>
         </item>
         <item>
          <widget class="QToolButton" name="btnMul2">
           <property name="text">
            <string>*2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnDiv2">
           <property name="text">
            <string>/2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>First line of ROI</string>
         </property>
         <property name="buddy">
          <cstring>sbRoiFirst</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="sbRoiFirst">
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Last line of ROI</string>
         </property>
         <property name="buddy">
          <cstring>sbRoiLast</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="sbRoiLast">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>31</number>
         </property>
         <property name="value">
          <number>31</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Frame integration time [Âµs]</string>
         </property>
         <property name="buddy">
          <cstring>dsbFrameTime</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="dsbFrameTime">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Temporal binning (accumulate subsequent frames)</string>
         </property>
         <property name="buddy">
          <cstring>cbBinning</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="cbBinning">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Acquire HW-correlator data</string>
         </property>
         <property name="buddy">
          <cstring>cbCorr</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="cbCorr">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Acquire raw data</string>
         </property>
         <property name="buddy">
          <cstring>cbCorr</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="cbRaw">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QLabel" name="labRuntime">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qfdoubleedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>cam_rh2v2_cfgdlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>cam_rh2v2_cfgdlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMul2</sender>
   <signal>clicked()</signal>
   <receiver>cam_rh2v2_cfgdlg</receiver>
   <slot>mul2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDiv2</sender>
   <signal>clicked()</signal>
   <receiver>cam_rh2v2_cfgdlg</receiver>
   <slot>div2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeSet1()</slot>
  <slot>mul2()</slot>
  <slot>div2()</slot>
 </slots>
</ui>
